src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -172,19 +172,21 @@ interface BootstrapMethodInvocation {
 
     /**
      * Gets the details for invoking a bootstrap method associated with the
-     * {@code CONSTANT_Dynamic_info} or {@code CONSTANT_InvokeDynamic_info} pool entry {@code cpi}
+     * {@code CONSTANT_Dynamic_info} or {@code CONSTANT_InvokeDynamic_info} pool entry
      * in the constant pool.
      *
-     * @param cpi a constant pool index
-     * @param opcode the opcode of the instruction that has {@code cpi} as an operand or -1 if
-     *            {@code cpi} was not decoded from an instruction stream
-     * @return the bootstrap method invocation details or {@code null} if the entry at {@code cpi}
+     * @param index if {@code opcode} is -1,  {@code index} is a constant pool index. Otherwise {@code opcode}
+     *              must be ${code Bytecodes.INVOKEDYNAMIC}, and {@code index} must be the operand of that
```

Comment:
`${code Bytecodes.INVOKEDYNAMIC}` -> `{@code INVOKEDYNAMIC}` (in numerous places)